src/eduid_userdb/logs/element.py

```diff
@@ -27,7 +27,9 @@ def __init__(self, created_by):
         :rtype: LogElement
         """
         self._required_keys = ['created_by', 'created_ts']
-        super(LogElement, self).__init__(data={'created_by': created_by, 'created_ts': True})
+        # Since log elements are already nearer dataclasses than the rest of elements,
+        # we do not deprecate direct calls to their __init__, which is close to what a dataclass would provide.
+        super(LogElement, self).__init__(data={'created_by': created_by, 'created_ts': True}, called_directly=False)
```

Comment:
remove the arguments to super() - not needed in python 3.7

Reply:
This one?